### ISSUE TITLE
Cut priorities

### ISSUE THREAD

**Issue by martinthomson** [martinthomson/http2v2#3]
This probably needs a skeleton left over so that we can avoid interoperability problems that might arise from interactions with peers that still have them.  There are some MUST-level requirements around parsing that would need to be identified and marked.

**Comment by yoavweiss** [comment]
I don't think we should cut priorities without a very strong reason, as they are successfully implemented by multiple implementers (even if for H3 we choose to go with a simpler scheme). Is there such a reason?
